storage/oasis/client.go

```diff
@@ -31,13 +36,19 @@ func NewConsensusClient(ctx context.Context, sourceConfig *config.SourceConfig)
 			network: sourceConfig.SDKNetwork(),
 		}, nil
 	}
+
+	// Create an API that connects to the real node, then wrap it in a caching layer.
 	var nodeApi nodeapi.ConsensusApiLite
 	nodeApi, err := history.NewHistoryConsensusApiLite(ctx, sourceConfig.History(), sourceConfig.Nodes, sourceConfig.FastStartup)
 	if err != nil {
 		return nil, fmt.Errorf("instantiating history consensus API lite: %w", err)
 	}
-	if sourceConfig.Cache != nil && sourceConfig.Cache.Consensus != nil {
-		nodeApi, err = file.NewFileConsensusApiLite(sourceConfig.Cache.Consensus.CacheDir, nodeApi)
+	if sourceConfig.Cache != nil {
+		cachePath := sourceConfig.Cache.CacheDir + "/consensus"
```

Comment:
Nit: Prefer `filepath.Join()` to avoid the risk of missing or duplicated `/`.
In fact, the current code sort-of assumes that `CacheDir` does not have a trailing `/` here, and then assumes that `CacheDir` does have one in `NewRuntimeClient()` :)

Reply:
Oops my bad, thanks for the catch! Updated